src/openzaak/conf/includes/base.py

```diff
@@ -171,6 +178,7 @@
     ),
     auto_display_default=False,
 )
+
```

Comment:
Unrelated to this line, but `NOTIFICATIONS_API_GET_DOMAIN` needs to be set as well to make sure `src/manage.py register_kanalen` works, should have the same value as the commonground-api-common setting